src/menu_item.c

```diff
@@ -8409,436 +8462,437 @@ void func_800A79F4(s32 arg0, char* arg1) {
+            case MENU_ITEM_TYPE_1AC: /* switch 8 */
+            case MENU_ITEM_TYPE_1AD: /* switch 8 */
+            case MENU_ITEM_TYPE_1AE: /* switch 8 */
+            case MENU_ITEM_TYPE_1AF: /* switch 8 */
+            case MENU_ITEM_TYPE_1B0: /* switch 8 */
+            case MENU_ITEM_TYPE_1B1: /* switch 8 */
+            case MENU_ITEM_TYPE_1B2: /* switch 8 */
+            case MENU_ITEM_TYPE_1B3: /* switch 8 */
+            case MENU_ITEM_TYPE_1B4: /* switch 8 */
+            case MENU_ITEM_TYPE_1B5: /* switch 8 */
+            case MENU_ITEM_TYPE_1B6: /* switch 8 */
+            case MENU_ITEM_TYPE_1B7: /* switch 8 */
+            case MENU_ITEM_TYPE_1B8: /* switch 8 */
+            case MENU_ITEM_TYPE_1B9: /* switch 8 */
+            case MENU_ITEM_TYPE_1BA: /* switch 8 */
+            case MENU_ITEM_TYPE_1BB: /* switch 8 */
+            case MENU_ITEM_TYPE_1BC: /* switch 8 */
+            case MENU_ITEM_TYPE_1BD: /* switch 8 */
+            case MENU_ITEM_TYPE_1BE: /* switch 8 */
+            case MENU_ITEM_TYPE_1BF: /* switch 8 */
+            case MENU_ITEM_TYPE_1C0: /* switch 8 */
+            case MENU_ITEM_TYPE_1C1: /* switch 8 */
+            case MENU_ITEM_TYPE_1C2: /* switch 8 */
+            case MENU_ITEM_TYPE_1C3: /* switch 8 */
+            case MENU_ITEM_TYPE_1C4: /* switch 8 */
+            case MENU_ITEM_TYPE_1C5: /* switch 8 */
+            case MENU_ITEM_TYPE_1C6: /* switch 8 */
+            case MENU_ITEM_TYPE_1C7: /* switch 8 */
+            case MENU_ITEM_TYPE_1C8: /* switch 8 */
+            case MENU_ITEM_TYPE_1C9: /* switch 8 */
+            case MENU_ITEM_TYPE_1CA: /* switch 8 */
+            case MENU_ITEM_TYPE_1CB: /* switch 8 */
+            case MENU_ITEM_TYPE_1CC: /* switch 8 */
+            case MENU_ITEM_TYPE_1CD: /* switch 8 */
+            case MENU_ITEM_TYPE_1CE: /* switch 8 */
+                func_800AF480(entry);
+                break;
+            case MENU_ITEM_UI_NONE:
+            case MENU_ITEM_UI_START_BACKGROUND:
+            case MENU_ITEM_UI_LOGO_AND_COPYRIGHT:
+            case MENU_ITEM_UI_PUSH_START_BUTTON:
+            case MENU_ITEM_UI_NO_CONTROLLER:
+                break;
+        }
+    }
+
+    for (j = 0; j < MENU_ITEM_UI_PRIORITY_MAX; j++) {
+        for (i = 0; i < MENU_ITEMS_MAX; i++) {
+            isRendered = false;
+            entry = &gMenuItems[i];
+            if (entry && entry) {} // ?
+            type = entry->type;
+            if ((type == MENU_ITEM_UI_NO_CONTROLLER) || (type == MENU_ITEM_UI_START_RECORD_TIME) ||
+                (type == MENU_ITEM_PAUSE)) {
+                if (priSpecial != 0) {
+                    isRendered = true;
+                }
+            } else if (priSpecial == 0) {
+                isRendered = true;
+            }
+            if ((isRendered != 0) && (j == (s8) entry->priority)) {
+                render_menus(entry);
+            }
+        }
+    }
+}
+
+void handle_menus_default(void) {
+    handle_menus_with_pri_arg(0);
```

Comment:
handle_menus_with_priority

what does 'handle' mean? Is this like Tick()? or spawn?